android/gltfio-android/src/main/java/com/google/android/filament/gltfio/AssetLoader.java

```diff
@@ -54,11 +53,21 @@
  *         assetLoader.createAssetFromBinary(ByteBuffer.wrap(bytes))!!
  *     }
  *
- *     ResourceLoader(engine).loadResources(filamentAsset).destroy()
+ *     val resourceLoader = ResourceLoader(engine)
+ *     resourceLoader.loadResources(filamentAsset)
+ *     for (uri in filamentAsset.resourceUris) {
+ *         val buffer = loadResource(uri)
+ *         resourceLoader.addResourceData(uri, buffer)
+ *     }
+ *     resourceLoader.destroy()
  *     animator = asset.getAnimator()
  *
  *     scene.addEntities(filamentAsset.entities)
  * }
+ *
+ * private fun loadResource(uri: String): Buffer {
+ *     TODO()
```

Comment:
Do we want to add a better sample implementation of this? I'm thinking that the URI might be anything pretty much so it'd be complicated to show all of it.

Reply:
It's fine not to show a complete implementation but instead of TODO we could write something like "// Load your asset here (e.g. using Android's AssetManager API)

Reply:
How about I add that as the comment in the `TODO()`? Or would you prefer a regular comment?

Reply:
Either way is fine.